firefox_ui_tests/remote/security/test_safe_browsing_warning_pages.py

```diff
@@ -81,12 +81,10 @@ def check_report_button(self, unsafe_page):
 
         # Wait for the button to become stale, then wait for page load
         # so we can verify the url even if a redirect happens
-        self.wait_for_condition(expected.element_stale(button))
+        Wait(self.marionette).until(expected.element_stale(button))
 
         # TODO: Bug 1140470: use replacement for mozmill's waitforPageLoad
-        self.wait_for_condition(lambda mn: mn.execute_script("""
-          return document.readyState == 'complete';
-        """))
+        Wait(self.marionette).until(lambda mn: mn.execute_script([...]))
```

Comment:
I gave an example of how it could look like. You cannot simply use `[...]` here but have to keep the original code. Please make sure to run all the tests also locally to verify your changes. This will cause a failure.